orc8r/tools/fab/vagrant.py

```diff
@@ -48,21 +48,36 @@ def setup_env_vagrant(machine='magma', apply_to_env=True, force_provision=False)
         # The machine isn't running. Most likely it's just not up. Let's
         # first try the simple thing of bringing it up, and if that doesn't
         # work then we ask the user to fix it.
-        print(
```

Comment:
Is it me or is `isUp` a confusing name on line 45? I tested it locally and it seems to evaluate to `False` if the VM is up and `True` if not, which is the opposite of what I would expect. In case it should be changed, I believe it should also be in snake case since it's a Python variable. See pylint output for that line:

`orc8r/tools/fab/vagrant.py:45:4: C0103: Variable name "isUp" doesn't conform to snake_case naming style (invalid-name)`

Reply:
It is confusing - I tried to be minimal invasive for a fix. But this is definitively worth a refactoring :) I will have a look

Reply:
added some refactoring